kubernetes-tests/src/test/java/io/fabric8/openshift/client/server/mock/OpenShiftVersionInfoTest.java

```diff
@@ -136,4 +136,95 @@ void testClusterVersioningOpenshift4() {
     assertEquals("4", versionInfo.getMajor());
     assertEquals("2.14", versionInfo.getMinor());
   }
+
+  @Test
+  void testClusterVersioningOpenshift4Unreleased() {
+    server.expect().get().withPath("/openshift/version").andReturn(404, "").always();
```

Comment:
Umm, Is this call required? I think default response from mockserver will be 404 if no expectation is set against this URL.

Reply:
Just copied from the existing test above

Reply:
Hmm, I just ran `OpenShiftVersionInfoTest#testClusterVersioningOpenshift4` and it passes even when I delete this. Could you please remove this from here and from the test above too?